lib/test_prof.rb

```diff
@@ -59,8 +59,10 @@ def asset_path(filename)
       ::File.expand_path(filename, ::File.join(::File.dirname(__FILE__), "..", "assets"))
     end
 
-    # Return a path to store artefact
-    def artefact_path(filename)
+    # Return a path to store artifact
+    def artifact_path(filename)
```

Comment:
😄 Actually, it's not a typo ([see](https://en.oxforddictionaries.com/definition/artefact)).

But I'm ok with the change, US variant is more popular.